lstmcpipe/stages/mc_process_dl1.py

```diff
@@ -416,7 +416,8 @@ def reprocess_dl1(
     slurm_account = batch_config["slurm_account"]
 
     if workflow_kind == "lstchain":
-        base_cmd = f"{source_environment} lstmcpipe_lst_core_dl1_dl1 -c {config_file} "
+        # TODO --no-image should be passed by default. To be configurable at lstmcpipe ?
+        base_cmd = f"{source_environment} lstmcpipe_lst_core_dl1ab --no-image -c {config_file} "
```

Comment:
nice catch !

Reply:
Do we want to make configurable this argument (at the `config_MC_prod.yml`). This would imply an arg more to manage. Not a problem indeed. I would just try to make it as easy as possible

Reply:
We can also leave it out I guess.
DL1ab processes dl1a files after all, so the images are somewhere already, right?

Reply:
(and we can also propagate the arg inside the script)

Reply:
> We can also leave it out I guess.
> DL1ab processes dl1a files after all, so the images are somewhere already, right?

Exactly. MC dl1 files (before merging) always contain images. Indeed, our `no_image: True` is just for the merging. 
So we could always run dl1ab from any previous MC prod because we are targeting the dl1 files (with images) before merging. 

However, the sub-dl1ab file will never contain images, so we won't be able to process dl1ab again from them (https://github.com/cta-observatory/cta-lstchain/pull/923#issuecomment-1044232723)

(wrapping up everything, it wasn't super clear in my head either)